### PR TITLE
Windowshade additions

### PR DESCRIPTION
**Pull request by knowuh**
Adds three new WindowShade types for IS2. 

* Framing the activity 
* Demo
* Offline activity 

PT Story:
https://www.pivotaltracker.com/story/show/168702283

Spec here: https://rivalf.axshare.com/#id=m50ia1&p=expanded_color_palette_and_add__icons&g=1

Branch deploy demo:
http://teacher-edition-tips-plugin.concord.org/branch/is2-windowshade-additions/demo.html

At some point we should try and refactor all the sass changes that happen across so many files.  

[#168702283]

### CODE REVIEW

File: src/components/_color-themes.sass [hunk 1]
```diff
@@ -2,14 +2,17 @@
 
 // Window shades have 5 different color themes based on the type:
```
Comment (kswenson) [inline src/components/_color-themes.sass]:
```suggestion
// Window shades have different color themes based on the type:
```

File: src/components/_color-themes.sass [hunk 2]
```diff
@@ -2,14 +2,17 @@
...
+// | theoryAndBackground | yellow    |
+// | teacherTip          | blue      |
+// | discussionPoints    | orange    |
+// | diggingDeeper       | purple    |
+// | howToUse            | aqua      |
+// | framing             | creamsicle|
+// | demo                | jewel     |
+// | offline             | green     |
+
 // Orthognal to each of these color sets based on the window shade type, each
```
Comment (kswenson) [inline src/components/_color-themes.sass]:
```suggestion
// Orthogonal to each of these color sets based on the window shade type, each
```

**Comment by knowuh** [comment]
@DALove1025  can I get your 👀 on this one?

**Comment by dougmartin** [comment]
LGTM :+1:

**Review by dougmartin — APPROVED** [review]
LGTM :+1:

**Review by DALove1025 — APPROVED** [review]
Reviewed w/ @knowuh. Looks good!

**Pull request merged**